cogs/quote_generator_cog/quote_generator_helper/image_creator_multi.py

```
@@ -22,120 +103,249 @@ def _compute_font_size(total_length: int) -> int:
     return 14
 
 
+def _fetch_image(url: str) -> Image.Image | None:
+    """Fetch a remote image as RGBA. Returns None on any failure."""
+    try:
+        resp = requests.get(url, timeout=5)
+        resp.raise_for_status()
+        return Image.open(BytesIO(resp.content)).convert("RGBA")
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)